README.md

```diff
@@ -1,23 +1,45 @@
 # Issue
 This is an attempt to compare performance between rxnetty and reactor netty http clients.
 
-# mockapplication
-mockapplication is a wiremock based application for stubbing http client benchmark use cases.
+# Building the common code
 
-# How to build 
-`mvn clean install` 
+This is the first thing you need to do.
 
-## IDE
-Run MockService 
-You could use mvn exec:java -pl mockapplication too.
+```sh
+mvn clean install
```

Comment:
Should we specify `-pl benchmarkcommon` here?

Reply:
I want to if we're removing the `performance` profile.  If not, I don't see much harm in leaving it as is.  Are you onboard with removing the profile?

Reply:
I would vote for the profile removal if there is not additional complexity in test setup/execution